__tests__/start.test.js

```diff
@@ -8,21 +8,25 @@ import { WALLET_CONSTANTS } from './integration/configuration/test-constants';
  * leaked connections at the end of the test.
  */
 
+const walletId = 'stub_wallet_start';
+
 describe('start api', () => {
-  beforeEach(async () => TestUtils.stopWallet());
+  beforeEach(async () => {
```

Comment:
Here we use `beforeEach` instead of the usual `beforeAll`/`afterAll` because we are testing the start wallet api